src/GlobalStyles.js

```diff
@@ -0,0 +1,20 @@
+import { createGlobalStyle } from 'styled-components'
+
+const GlobalStyles = createGlobalStyle`
+
```

Comment:
Don't you want to add `box-sizing: border-box;` as a default for every component?
```suggestion
* {
  box-sizing: border-box;
  }
```